### ISSUE TITLE
How to calculate MOS_zscore in koniq10k datasets?

### ISSUE THREAD

**Issue by wangbin2018** [SSL92/hyperIQA#15]
such as c1 = 0, c2 = 0, c3 = 25, c4 = 73, c5 = 7, c_total = 105, MOS = 3.83, SD = 0.53

**Comment by SSL92** [comment]
In our downloaded score file, there's one column 'MOS_zscore' attached behind column SD, maybe you could refer to the published dataset to see if there's any update.

**Comment by wangbin2018** [comment]
Yes, MOS_zscore = 77.38362, but I don not know how to calculate this Mark.
Another datasets maybe only give MOS mark，So I want to understand.
Thank for your replay！

**Comment by SSL92** [comment]
This might be the post-processed MOS score, you can refer to their [origin paper](https://arxiv.org/pdf/1910.06180v1.pdf) for an answer.

**Comment by wangbin2018** [comment]
I read this paper already，but only got how to calculate MOS，not MOS_zscore.
Maybe I didn't see it O.O

**Comment by ilovetlic** [comment]
image_name: the image file name
c1-c5: number of ratings for each ACR value 
c_total: total number of judgments per image
MOS: Mean Opinion Scores of the 5-point ACR
​SD: ​Standard Deviation of the MOS
MOS_zscore: ACR scores are first normalised for each user by z-scoring the user ratings

**Comment by anirban-ivan** [comment]
Hi, I am confused about the same issue. @wangbin2018 did you finally understand on how to calculate the MOS_zscore?
